test/regression_tests/ocean_large_eddy_simulation_regression_test.jl

```diff
@@ -106,21 +106,23 @@ function run_ocean_large_eddy_simulation_regression_test(arch, closure)
 
     solution₀, Gⁿ₀, G⁻₀ = get_fields_from_checkpoint(initial_filename)
 
+    Nz = grid.Nz
+
     model.velocities.u.data.parent .= ArrayType(solution₀.u)
     model.velocities.v.data.parent .= ArrayType(solution₀.v)
-    model.velocities.w.data.parent .= ArrayType(solution₀.w)
+    model.velocities.w.data.parent[:, :, 1:Nz+2] .= ArrayType(solution₀.w)
```

Comment:
Hmmm yeah the regression test changes slightly but perhaps it's not worth recreating regression files as the numbers themselves have not changed?

Reply:
I was thinking that for situations like this, the right approach is 

1) change the regression test so it passes, thereby verifying expected behavior; then
2) submit a subsequent PR that recreates the regression files.

We don't want to make the changes that break the regression test *and* recreate regression files in the same PR, because then we can make a mistake and subvert the purpose of the regression test (which is to prevent unexpected changes to proper functioning of the model). 

In this change, its clear from the commit that the needed change is only to the view on which the data is compared; otherwise the regression test is unchanged. This verifies that the regression test *should* pass.